KubeArmor/monitor/processTree.go

```diff
@@ -131,6 +134,31 @@ func (mon *SystemMonitor) GetExecPath(containerID string, pid uint32) string {
 	return ""
 }
 
+// GetProcessName Function
+func (mon *SystemMonitor) GetProcessName(containerID string, pid uint32) string {
+	ActivePidMap := *(mon.ActivePidMap)
+	ActivePidMapLock := *(mon.ActivePidMapLock)
+
+	ActivePidMapLock.RLock()
+	defer ActivePidMapLock.RUnlock()
+
+	if pidMap, ok := ActivePidMap[containerID]; ok {
+		if node, ok := pidMap[pid]; ok {
+			return string(bytes.Trim([]byte(node.Comm), "\x00"))
+		}
+	}
+
+	return ""
+}
+
+// GetProcessPathfromProc function
+func (mon *SystemMonitor) GetProcessPathFromProc(pid uint32) string {
+	procPath := fmt.Sprintf("/proc/%d/exe", pid)
+	path, _ := os.Readlink(procPath)
```

Comment:
Won't this cause infinite loop when file visibility is enabled? @nam-jaehyun 

On any file open kubearmor gets a SYS_OPENAT event -> kubearmor tries to get the process path from the proc fs -> opening file in procfs results in another SYS_OPENAT event